models/players.js

```diff
@@ -30,6 +35,20 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var formatEvenOdds = function(players){
```

Comment:
I was just gonna do it with css, if the screen is a certain width, then I will disable the float that the class "right" will have